core/script_language.cpp

```diff
@@ -602,3 +603,7 @@ PlaceHolderScriptInstance::~PlaceHolderScriptInstance() {
 		script->_placeholder_erased(this);
 	}
 }
```

Comment:
I think I wouldn't add this here, since SceneTree should not be accessed from core..
maybe we should simply move the remote debugger to scene and set it in there..